src/StaticFAQ/ArticleFeedback/ScreenInput.js

```diff
@@ -88,8 +88,14 @@ class ScreenInput extends React.Component<Props, State> {
         backgroundColor="#f5f7f9"
       >
         <form onSubmit={this.handleSubmit}>
-          <div className="close-icon">
-            <Close fill="#bac7d5" height="12" onClick={this.closeScreen} />
+          <div
+            className="close-icon"
+            onClick={this.closeScreen}
+            onKeyUp={this.closeScreen}
```

Comment:
Maybe we can just `onKeyUp={null}`? This would clarify that this listener is added just in order to pass lint rules and serves no actual purpose.